prod/docker-compose.yml

```diff
@@ -28,8 +28,17 @@ services:
     # Log printing via Rich is enhanced by a TTY
     tty: true
     env_file: ./.env
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
     volumes:
       - "${MIQA_MOUNT_DIR}:${MIQA_MOUNT_DIR}"
+      - "/home/anne/miqa/samples:/mnt/mydata"
+      - "/home/anne/miqa/samples:/srv/samples"
```

Comment:
These should be relative paths

Reply:
I meant to leave this whole file out of the commit, whoops. I'll fix it.

Reply:
I realize I had committed it because of the nvidia driver change, so I reconciled that change with removing my local-fix changes. Now this file does not change the postgres port or the django mounted volumes, but does change the GPU resource for django.